workers/ohsome_quality_analyst/indicators/models.py

```diff
@@ -45,13 +45,20 @@ class Result(BaseModel):
 
     description: str
     html: str
-    timestamp_oqt: datetime = datetime.now(timezone.utc)  # UTC datetime object
-    timestamp_osm: datetime | None = None
+    timestamp_oqt: datetime = Field(
+        default=datetime.now(timezone.utc), alias="timestampOQT"
+    )  # UTC datetime object
+    timestamp_osm: datetime | None = Field(default=None, alias="timestampOSM")
     value: float | None = None
     class_: Literal[1, 2, 3, 4, 5] | None = None
     figure: dict | None = None
     svg: str | None = None
 
+    class Config:
+        alias_generator = snake_to_lower_camel
+        extra = "forbid"
```

Comment:
```suggestion
```

Reply:
not necessary, as `data` is included outside of this model